src/Tools/MicroBuild/publish-assets.ps1

```diff
@@ -43,6 +43,7 @@ try
         "dev15.0.x" { } 
         "master" { } 
         "dev_pilchie_Fix389698" { }
+        "features/readonly-ref" { }
```

Comment:
This won't work because of line 39 above. You need to pull more recent bits from master branch